OloEngine/src/OloEngine/Renderer/Font.cpp

```
@@ -277,4 +279,93 @@ namespace OloEngine
         }
         return {};
     }
+
+    f32 Font::MeasureLine(std::string_view line, f32 fsScale, f32 kerning) const
```

Cppcheck (2.20.0)

[style] The function 'MeasureLine' is never used.

(unusedFunction)